ddtrace/internal/symbol_db/symbols.py

```diff
@@ -203,6 +204,11 @@ def _(cls, module: ModuleType, data: ScopeData):
             except Exception:
                 log.debug("Cannot get child scope %r for module %s", child, module.__name__, exc_info=True)
 
+        source_bytes = module_origin.read_bytes()
+        source_git_hash = sha1()
+        source_git_hash.update(f"blob {len(source_bytes)}\0".encode())
+        source_git_hash.update(source_bytes)
```

Comment:
Is this safe enough? Should we have a maximum number of bytes, and read in chunks rather than store the full file in memory? Should we define an "error" for `file_hash` or is empty good enough?